scripts/cluster/create_multinode_cluster.go

```diff
@@ -0,0 +1,177 @@
+package cluster
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	configs "github.com/vhive-serverless/vHive/scripts/configs"
+	utils "github.com/vhive-serverless/vHive/scripts/utils"
+)
+
+func CreateMultinodeCluster(stockContainerd string) error {
+	// Original Bash Scripts: scripts/cluster/create_multinode_cluster.sh
+
+	err := CreateMasterKubeletService()
+	if err != nil {
+		return err
```

Comment:
```suggestion
	if err := CreateMasterKubeletService(); err != nil {
		return err
```

refactor for brevity here and below